apps/prairielearn/src/batched-migrations/20250502181707_json_columns__backfill.ts

```diff
@@ -11,7 +11,11 @@ import { syncDiskToSqlWithLock } from '../sync/syncFromDisk.js';
 
 export default makeBatchedMigration({
   async getParameters() {
-    const result = await queryRow('SELECT MAX(id) as max from pl_courses;', {}, z.string());
+    const result = await queryRow(
+      'SELECT MAX(id) as max from pl_courses;',
+      {},
+      z.string().nullable(),
+    );
```

Comment:
```suggestion
    const result = await queryRow(
      'SELECT MAX(id) as max from pl_courses;',
      z.string().nullable(),
    );
```
Nit: `queryRow` does not require the second argument if it's empty.

Reply:
I think there was an issue where this was applied twice, and the second one deleted the wrong line?